backend/core/src/Core.Infrastructure/Compliance/SendGridMailService/MailTemplate.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2023 Quantoz Technology B.V. and contributors. Licensed
+// under the Apache License, Version 2.0. See the NOTICE file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+using Newtonsoft.Json;
+
+namespace Core.Infrastructure.Compliance.SendGridMailService
+{
+    public class MailTemplate
+    {
+        [JsonProperty("customerFullName")]
+        public string? CustomerFullName { get; set; }
+
+        [JsonProperty("amount")]
+        public string? Amount { get; set; }
+
+        [JsonProperty("accountCode")]
+        public string? AccountCode { get; set; }
+
+        [JsonProperty("customerBankAccount")]
+        public string? BankAccount { get; set; }
+
+        [JsonProperty("transactionCode")]
+        public string? TransactionCode { get; set; }
+
+        [JsonProperty("payoutAmount")]
+        public string? PayoutAmount { get; set; }
```

Comment:
What is amount, payoutamount and shouldn't there also be fundingamount?

Reply:
Amount - Requested Amount
Payout Amount - Net fiat amount
As per Gaston, we don't need to show the netFundingAmount to the customer right now.